src/state/configure-store.ts

```diff
@@ -5,6 +5,9 @@ import { createLogicMiddleware } from "redux-logic";
 
 import { enableBatching, initialState, metadata, selection, State } from "./";
 import RequestClassToUse from "./image-dataset";
+import devtoolsCompose from "./redux-devtools";
+
+const ENABLE_DEVTOOLS = false;
```

Comment:
it seems like this should be set by the npm start command. we already have an `npm start:dev-db` we could have an `npm start:dev` that sets both the db and turns on these dev tools.

Reply:
I agree w/that suggestion, we can feed this in as a build-time configuration parameter